draft-how-does-mutant-work.markdown

```diff
@@ -1392,14 +1391,18 @@ def run_driver(driver)
 
   status.payload
 end
-```    
+```
 
 Ok, that's conceptually simple: 
 
 * The `driver` that was returned by `Parallel.async(mutation_test_config)` is running our specs asynchronously and in parallel
 * We loop continuously over the `driver` and check its status
 * As soon as the `driver` is done, i.e. all mutants have either been killed or have survived, we return the result to the reporter and exit
 
+When it comes to how `Mutant` does finally run your tests against mutations in parallel I barely touched the surface. `Mutant` uses its own framework for parallelism and implements its own actor model on top of that.
```

Comment:
Its the other way round, mutant has an actor that backs parallel, not parallel backs actors.